test/integration/lib/storages/RedisProxyCache.int.test.ts

```diff
@@ -41,7 +43,12 @@ const redisClusterProxyConfig = fixtures.redisClusterProxyConfigDto({
 logger.info('redis proxyConfigs', { redisClusterProxyConfig, redisProxyConfig });
 
 describe('RedisProxyCache Integration Tests -->', () => {
-  const runUseCases = (proxyCache: IProxyCache, anotherProxyCache: IProxyCache) => {
+  const runUseCases = (proxyCache: IProxyCache<RedisClientType>, anotherProxyCache: IProxyCache<RedisClientType>) => {
```

Comment:
Why do we need to specify a particular clients here? The idea is `runUseCases` will be used by any proxyCache impl., not only redis-related